x/epochs/keeper/epoch.go

```diff
@@ -59,35 +60,27 @@ func (k Keeper) AddEpochInfo(ctx sdk.Context, epoch types.EpochInfo) error {
 
 // UpsertEpochInfo inserts the epoch if does not exist, and overwrites it if it does.
 func (k Keeper) UpsertEpochInfo(ctx sdk.Context, epoch types.EpochInfo) {
-	store := ctx.KVStore(k.storeKey)
-	value, err := proto.Marshal(&epoch)
-	if err != nil {
-		panic(err)
-	}
-	store.Set(append(types.KeyPrefixEpoch, []byte(epoch.Identifier)...), value)
+	k.Epochs.Insert(ctx, epoch.Identifier, epoch)
```

Comment:
Are these methods of any value? In my opinion, any function whose implementation is one line long can be replaced with executing that actual line.

Reply:
they have a value in some specific cases, probably not on inserting, but when we want to define expected_keepers interface.

Imagine that you want to get the Epoch with ID x, you cannot define the Epochs type in an interface. 

there is a thin line on when to do it and when not.

The ideal would be that a keeper only have the exported methods it is allowed to be called from other modules and they do not break your data in unexpected ways.

Reply:
As it stands right now, other keepers can directly access the collection fields of this keeper right?

Reply:
depends if you use the pattern of expected keepers or not